### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by YuliaHladyshkevych**

### CODE REVIEW

**Review by petrykivd — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
 
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(customers: list, hall_number: int,
+                 cleaner: str, movie: str) -> None:
+
+    customers = [Customer(customer["name"], customer["food"])
+                 for customer in customers]
```
[the remark this answers is not in the document]
Comment (LiudmylaKulinchenko) [reply app/main.py]: @petrykivd You don't need to use it everywhere, it is not a bad practice in all cases
In this case you don't want Customer to have name or food equal to `None`, so it is better to refer by key directly

File: app/main.py [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+from app.people.cinema_staff import Cleaner
 
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(customers: list, hall_number: int,
+                 cleaner: str, movie: str) -> None:
+
+    customers = [Customer(customer.get("name"), customer.get("food"))
```
Comment (LiudmylaKulinchenko) [inline app/main.py:10]: In this case you don't want Customer to have name or food equal to None, so it is better to refer by key directly

File: app/main.py [hunk 3]
```diff
@@ -1,6 +1,21 @@
...
+    cinema_hall = CinemaHall(hall_number)
+    cinema_cleaner = Cleaner(cleaner)
+
+    for customer in customers:
+        CinemaBar.sell_product(customer.food, customer)
+
+    cinema_hall.movie_session(
+        movie_name=movie,
+        customers=customers,
+        cleaning_staff=cinema_cleaner)
```
Comment (LiudmylaKulinchenko) [inline app/main.py:21]: Fix formatting: brackets must be located in one of two options: open and start with the text or have a line break between the text

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

**Review by LiudmylaKulinchenko — APPROVED** [review]
Well done! 👏☀🐣

**Pull request closed without merging**